src/api/config.js

```diff
@@ -3,20 +3,20 @@ const upath = require('upath');
 const fs = require('fs-extra');
 const editJsonFile = require('edit-json-file');
 
-const getFullConfig = async (path) => {
+const getFullConfig = async path => {
```

Comment:
I am super confused why github is removing these.. it appearently is valid js though

Reply:
Oh its from the linter, I have github hooks set up so the linter runs on commit 👍